src/main/java/frc/robot/notification/ILedIO.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.notification;
+
+public interface ILedIO {
+  
+  public void setSpark(double sparkValue);
+
+  public boolean getIsAlive(); 
```

Comment:
is this relevant to the operation of the robot? Or just for debugging? Can we come up with examples of when this would be the data point that solved a problem?

Reply:
The boolean method is useful for if we determine there to be a connection issue with the blinkin'. When running the simulator we just return true, but when enabled this will return the connection status of the blinkin' over the PWM channel.